### DISCUSSION TITLE
should the bot be admin?

### DISCUSSION THREAD

**Discussion by s-ahmadzadeh** [MrMissx/Telegram_Forwarder#16] [FAQ] [answered]
I want to send messages from a specific channel to another channel (my own channel), is this necessary to add bot to both of the channels?
I can not add the bot as an admin to the source channel (from), but I can add the bot as an admin to the destination channel (to) which is my own channel

**Comment by MrMissx** [comment, marked as answer]
yeahh...
To be clear, the bot need to be in both TO_CHAT and FROM_CHAT and the bot need a message access to the chat (Admin if it's a channel) unless it wouldn't work.
